deploy/charts/rook-ceph/values.yaml

```
@@ -652,6 +652,13 @@ enableOBCWatchOperatorNamespace: true
 # @default -- `ceph cluster namespace`
 obcProvisionerNamePrefix:
 
+# -- Many OBC additional config fields are dangerous for administrators to allow users control over.
+# The safe and default-allowed fields are 'maxObjects' and 'maxSize'.
+# Other fields should be considered risky. To allow all additional configs, use this value:
+#   "maxObjects,maxSize,bucketMaxObjects,bucketMaxSize,bucketPolicy,bucketLifecycle,bucketOwner"
+# @default -- "maxObjects,maxSize"
+# obcAllowAdditionalConfigFields: "maxObjects,maxSize"
```

What about specifying the default directly? IMO it helps clarify the default.
```suggestion
obcAllowAdditionalConfigFields: "maxObjects,maxSize"
```

---

Will do. I'm going to have to do this for the doc generation regardless